lib/fluent/plugin/in_udp.rb

```diff
@@ -40,6 +40,8 @@ class UdpInput < Input
     config_param :message_length_limit, :size, default: 4096
     desc "Remove newline from the end of incoming payload"
     config_param :remove_newline, :bool, default: true
+    desc "The max size of socket receive buffer. SO_RCVBUF"
+    config_param :recv_buffer, :size, default: nil
```

Comment:
`receive_buffer_size` is better.

Reply:
Okay. I just followed `recv_timeout`, but it isn't a parameter name yet.

Reply:
Yeah, public name should be clear.